clients/tabby-agent/src/AgentConfig.ts

```diff
@@ -70,6 +77,11 @@ export const defaultAgentConfig: AgentConfig = {
       experimentalStripAutoClosingCharacters: false,
       maxPrefixLines: 20,
       maxSuffixLines: 20,
+      experimentalDeclarations: {
```

Comment:
```suggestion
      fillDeclarations: {
```

This won't go through a experimental launch approach, just name it as ordianry flag.